lib/requests/request_executor.rb

```diff
@@ -165,6 +168,8 @@ def execute_command(command, server_node)
       
       begin
         response = http_client(server_node).request(request)
+      rescue Net::OpenTimeout => ssl_exception
```

Comment:
This begin block doesn't look good. It seems you catch same exception, so that 
```
       rescue Net::OpenTimeout => timeout_exception
          request_exception = timeout_exception
```

never gets called. Am I right @a-serdukov-mobilunity ?

Reply:
yes it just a misspell in code. OpenSSL error should be catched here